geometry_msgs/Point center      # Central point [m]
geometry_msgs/Vector3 velocity  # Linear velocity [m/s]
float64 radius                  # Radius [m]

string obstacle_id              # ID of the obstacle
bool tracked                    # If the obstacle is tracked
